src/components/Receiving/ItemDetails.js

```diff
@@ -100,115 +149,58 @@ class ItemDetails extends Component {
-            defaultValue={<FormattedMessage id="ui-orders.receiving.itemStatus.received" />}
-            fullWidth
-            selectClass={css.itemStatusField}
-          >
-            {Object.keys(ITEM_STATUS).map((key) => (
-              <FormattedMessage
-                id={`ui-orders.receiving.itemStatus.${key}`}
-                key={key}
-              >
-                {(message) => <option value={ITEM_STATUS[key]}>{message}</option>}
-              </FormattedMessage>
-            ))}
-          </Select>
-        </div>
-      ),
-    });
-  }
-
   render() {
-    const { close } = this.props;
-    const { allChecked, checkedItems, currentLine, lineItems } = this.state;
-    const poLineIdsList = Object.keys(lineItems);
+    const { close, locationsOptions, linesItemList } = this.props;
+    const { allChecked, currentLine, lineItems } = this.state;
+    const poLineIdsList = Object.keys(linesItemList);
     const poLineId = poLineIdsList[currentLine];
     const poLineNumber = get(lineItems, [poLineId, 0, 'poLineNumber'], '');
     const title = get(lineItems, [poLineId, 0, 'title'], '');
+    const isReviewScreen = currentLine >= poLineIdsList.length;
 
     return (
-      <Modal
-        id="data-test-piece-details-modal"
-        label={
-          (currentLine >= poLineIdsList.length)
-            ? <FormattedMessage id="ui-orders.receiving.reviewDetails" />
-            : <FormattedMessage id="ui-orders.receiving.modalPaneTitle" values={{ poLineNumber, title }} />
-        }
-        footer={
-          <ItemDetailsFooter
-            close={close}
-            onClickNext={this.onClickNext}
-            onClickPrevious={this.onClickPrevious}
-            poLineIdsListLenght={poLineIdsList.length}
-            currentLine={currentLine}
-            checkedItemsListLength={checkedItems.length}
-          />
-        }
-        open
-      >
-        {(this.state.currentLine > poLineIdsList.length)
-          ? <div>Receive Details</div>
-          : (
-            <MultiColumnList
-              contentData={lineItems[poLineId]}
-              formatter={this.getResultsFormatter()}
-              visibleColumns={['isChecked', 'barcode', 'format', 'location', 'itemStatus']}
-              columnMapping={{
-                isChecked: <Checkbox type="checkbox" checked={allChecked[poLineId]} onChange={() => this.toggleAll(poLineId)} />,
-                barcode: <FormattedMessage id="ui-orders.receiving.barcode" />,
-                format: <FormattedMessage id="ui-orders.receiving.format" />,
-                location: <FormattedMessage id="ui-orders.receiving.location" />,
-                itemStatus: <FormattedMessage id="ui-orders.receiving.itemStatus" />,
-              }}
-              columnWidths={{
-                isChecked: '5%',
-                barcode: '25%',
-                format: '20%',
-                location: '35%',
-                itemStatus: '15%',
-              }}
+      <div data-test-item-details>
```

Comment:
do you need this if your modal has id and could be referenced by ID in tests? IMHO, DOM is cleaner...

Reply:
Not now, it was written before you made updates with id.